# special string for an unnamed agent
string NO_AGENT_NAME = NoName

# name of the agent who is not ready
string agent_name

# action agent is not ready for
string action

# commanded task that action is part of
string commanded_task
